evm/test/TrimmedAmount.t.sol

```diff
@@ -148,12 +159,62 @@ contract TrimmingTest is Test {
         uint8 targetDecimals = 6;
         uint256 amount = 5 * 10 ** sourceDecimals;
 
-        TrimmedAmount memory trimmedAmount = amount.trim(sourceDecimals, 8);
+        TrimmedAmount trimmedAmount = amount.trim(sourceDecimals, 8);
         // trimmed to 8
         uint256 amountRoundTrip = trimmedAmount.untrim(targetDecimals);
         // untrim to 6
         uint256 expectedRoundTrip = 5 * 10 ** targetDecimals;
 
         assertEq(expectedRoundTrip, amountRoundTrip);
     }
+
+    function testFuzz_AddOperatorOverload(TrimmedAmount a, TrimmedAmount b) public {
+        vm.assume(a.getDecimals() == b.getDecimals());
+
+        // check if the add operation reverts on an overflow.
```

Comment:
this comment doesn't line up with the the test -- we're not checking for overlows here or any revert behaviour -- instead we're only testing non-overflowing cases

Reply:
the line after provides more context, its checking if the sum overflows u64. if so, the test case is discarded via vm.assume; therefore, this test case checks non-overflowing cases